src/base/document.js

```diff
@@ -0,0 +1,109 @@
+import { DocumentsManager } from '../api2'
```

Comment:
`/base` is a folder ONLY for core-related stuff, it shouldn't be here

Reply:
moved to src/utils (both document and upload-documents)